### ISSUE TITLE
[Texteditor]: WCAG and other issues

### ISSUE THREAD

**Issue by ivarkallejarv** [TEHIK-EE/tedi-design-system#808]
## Problem

- Use TextEditor component
- TextEditor text input is missing accessible name
- Formatting buttons (bold/italic/bullet) are missing accesible texts

![Image](https://github.com/user-attachments/assets/226468c2-6640-4c45-9792-181d5ea5db07)

**Comment by tobre6** [comment]
Problem confirmed. Will be resolved when component is moved to TEDI-READY.

Current suggestion: Try to fix in the project, maybe manipulating DOM with javascript and add aria-label attributes to buttons.
